main.js

```diff
@@ -65,6 +65,167 @@ let startupFinishedObserver = null;
 let hotkey;
 let VerticalTabsWindowId = 1;
 
+function reminderTour(win){
+  win.reminderTour = true;
+  set('extensions.tabcentertest1@mozilla.com.doNotShowTour', true); //after reminder tour shown once, we will not show again.
+  firstInstallTour(win);
+}
+function firstInstallTour(win) {
+  if (win.activeInstall || win.reminderTour) {
+    let details = {};
+    if (get('extensions.tabcentertest1@mozilla.com.tourComplete')) {
+      details.tour_type = 'completed_reminder';
+    } else if (win.reminderTour){
+      details.tour_type = 'reminder';
+    } else {
+      details.tour_type = 'install';
+    }
+    utils.sendPing('tour_began', win, details);
+
+    win.activeInstall = false;
+    let document = win.document;
+    let sidetabsbutton = document.getElementById('side-tabs-button');
+    let panel = document.createElement('panel');
+    let outerbox = document.createElement('vbox');
+    let instructions = document.createElement('description');
+    let progressButton = document.createElement('button');
+    let tourTitle = document.createElement('h2');
+    let tourVideoWrapper = document.createElement('div');
+    tourVideoWrapper.setAttribute('id', 'tour-video-wrapper');
+    let tourVideo = document.createElementNS('http://www.w3.org/1999/xhtml', 'video');
+    tourVideo.setAttribute('src', self.data.url('Intro.mp4'));
+    tourVideo.setAttribute('autoplay', 'true');
+    tourVideo.setAttribute('loop', 'true');
```

Comment:
(Possibly even extend it to take a list of children as an extra parameter, and append them to the newly-created element.)